pkg/kotsadm/main.go

```diff
@@ -116,6 +117,12 @@ func Upgrade(clientset *kubernetes.Clientset, upgradeOptions types.UpgradeOption
 		return errors.Wrap(err, "failed to read deploy options")
 	}
 
+	// If user has passed in the flag to migrate minio, save this status as part of the install
+	// Only works if they have not already chose to opt-in
+	if !upgradeOptions.IncludeMinio && deployOptions.IncludeMinioSnapshots {
```

Comment:
i guess this is not valid now, right? since they're both using the same flag

Reply:
`deployOptions` contains the value from the configmap if they previously opt-ed in. My intent was trying to prevent folks from going backward and specifying `with-minio=true`.